src/components/settings/SpendingLimits/NewSpendingLimit/steps/SpendingLimitForm.tsx

```diff
@@ -110,9 +110,9 @@ export const SpendingLimitForm = ({ data, onSubmit }: Props) => {
               {...register('amount', {
                 required: true,
                 validate: (val) =>
-                  validateAmount(val) ||
-                  validateDecimalLength(val, selectedToken?.tokenInfo.decimals) ||
-                  validateUpperSpendingLimitAmount(val),
+                  validateAmount(val) || Number(val) < 1
```

Comment:
Why does the < 1 check has to be here? Can't you do it in `validateDecimalLength` and `validateUpperSpendingLimitAmount`?

Reply:
Good point. I will have to rework `validateUpperSpendingLimitAmount` as it doesn't like decimals at all.